package.json

```diff
@@ -1,14 +1,15 @@
 {
     "dependencies": {
-        "@demox-labs/solrand": "^0.0.3",
-        "@project-serum/anchor": "^0.18.2",
-        "@project-serum/anchor-cli": "0.18.2"
+        "@demox-labs/solrand": "^0.0.5",
```

Comment:
Not about this file, but should the `package-lock.json` be completely removed?

Reply:
Good catch, it should probably be added back in here